### ISSUE TITLE
runserver.py: error: argument -k/--gmaps-key is required

### ISSUE THREAD

**Issue by rynsey** [AHAAAAAAA/PokemonGo-Map#3433]
## Expected Behavior
I have entered my API key in the line gmaps-key & removed the # from the beginning yet i always get this error upon running the script.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by Rightbrained** [comment]
When you're entering you're run command you've still got the -k in there, take that out and it should work

**Comment by nurdiansyah01** [comment]
enter this
"C:\Python27\python.exe" runserver.py -a ptc -u nurdiansyah -p nurdin933 -l "34.010808, -118.495793" -st 10 -H 0.0.0.0 -L en -k "API KEY"

**Comment by nurdiansyah01** [comment]
"C:\Python27\python.exe" runserver.py -a ptc -u username -p password -l "koordinate/location" -st 10 -H 0.0.0.0 -L en -k "API KEY"

**Comment by A5LAN** [comment]
yeah get ur api key from here https://developers.google.com/maps/documentation/javascript/get-api-key

**Comment by Rightbrained** [comment]
Guys... he says he has his API key and put it into the Config.ini 
Running '"C:\Python27\python.exe" runserver.py -a ptc -u username -p password -l "koordinate/location" -st 10 -H 0.0.0.0 -L en -k "API KEY"' will net him a lot of errors if he has already defined the user name, password, location, and as he stated, API key.

**Comment by Cetacea35** [comment]
As another noob, who just installed stuff and had that same problem, I can tell you what you need to do is change that file name from config.ini.example to just config.ini

**Comment by antonio081014** [comment]
You could also just append the key as the argument followed by `-k YOUR_KEY`.
If you need to know where to get the key, please read WIKI first.

**Comment by MrKetchumAll** [comment]
Make sure to use a different account to make the key, just to be on the safe side.
